### ISSUE TITLE
Error in libraries syntax when loading with latest nightly build (13 Sep. 2016)

### ISSUE THREAD

**Issue by mpkopec** [KiCad/kicad-library#697]
Hello everyone,
I have found that when trying to open eeschema project an error occurs, which seems to be related to the syntax of the library. Below you can find an image with the error message. Did anyone stumble upon this type of error and if so how to deal with it?

![kicad-error_loading-libs](https://cloud.githubusercontent.com/assets/8792915/18474836/4adc9d06-79c4-11e6-910e-767153859552.png)

**Comment by SchrodingersGat** [comment]
Can you post a copy of your "cmos_ieee.lib" file?

**Comment by mpkopec** [comment]
Of course, you can find it below. I have just one more remark - this is freshly cloned from github.

[cmos_ieee.zip](https://github.com/KiCad/kicad-library/files/469892/cmos_ieee.zip)

**Comment by mpkopec** [comment]
Ok, the problem does not exist in the stable version, so it's the KiCad itself problem apparently.
